src/App.jsx

```diff
@@ -1,60 +1,178 @@
-import "./App.css"
+import { useState } from "react";
+import "./App.css";
 
 function App() {
+  const [panelOne, setPanelOne] = useState("0");
+  const [panelTwo, setPanelTwo] = useState("0");
+  const [panelMath, setPanelMath] = useState("+");
+  const [answer, setAnswer] = useState(0);
+
+  const addToPanelOne = (event) => {
+    console.log(event.currentTarget.value);
+    console.log("panel one is", panelOne);
+    if (panelOne === "0") {
+      setPanelOne(event.currentTarget.value);
+    } else if (event.currentTarget.value === "Clear") {
+      setPanelOne("0");
+    } else {
+      setPanelOne(panelOne.toString() + event.currentTarget.value.toString());
+    }
+  };
+
+  const addToPanelMath = (event) => {
```

Comment:
not sure what panelMath represents? is it the operation? `setOperation` might be a better name